iOS/README.rst

```diff
@@ -0,0 +1,301 @@
+====================
+Python on iOS README
+====================
+
+:Authors:
+    Russell Keith-Magee (2023-11)
+
+This document provides a quick overview of some iOS specific features in the
+Python distribution.
+
+These instructions are only needed if you're planning to compile Python for iOS
+yourself. Most users should *not* need to do this. If you're looking to
+experiment with writing an iOS app in Python, tools such as `BeeWare's Briefcase
+<https://briefcase.readthedocs.io>`__ and `Kivy's Buildozer
+<https://buildozer.readthedocs.io>`__ will provide a much more approachable
+user experience.
+
+Compilers for building on iOS
+=============================
+
+Building for iOS requires the use of Apple's Xcode tooling. It is strongly
```

Comment:
I really like this README overall and I like the explicit statement about using Apple tooling. I think this section should also explicitly state whether Apple's Command Line Tools are sufficient for building and/or testing (which is the case for macOS builds) or, if a full Xcode install is needed and, if so, what optional components are needed.

Reply:
I've added a note to this effect.